gemrb/plugins/SDLVideo/SDL20Video.cpp

```diff
@@ -58,29 +62,26 @@ int SDL20VideoDriver::CreateDriverDisplay(const Size& s, int bpp, const char* ti
 	Log(MESSAGE, "SDL 2 Driver", "Creating display");
 	// TODO: scale methods can be nearest or linear, and should be settable in config
 	SDL_SetHint(SDL_HINT_RENDER_SCALE_QUALITY, "nearest");
-	//SDL_SetHint(SDL_HINT_RENDER_VSYNC, "0");
 
-#if OPENGL_BACKEND
-#if OPENGL_BACKEND == OpenGL
+#if USE_OPENGL_BACKEND
+#if USE_OPENGL_API
 	const char* driverName = "opengl";
-#elif
+#elif USE_GLES_API
 	const char* driverName = "opengles2";
 #endif
 	SDL_SetHint(SDL_HINT_RENDER_DRIVER, driverName);
-
-	/*
-	 these have no effect on the context created by SDL_CreateRenderer
-	   SDL_GL_SetAttribute(SDL_GL_CONTEXT_FLAGS, SDL_GL_CONTEXT_FORWARD_COMPATIBLE_FLAG);
-	   SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_CORE);
-	   SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, 3);
-	   SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 2);
-	*/
 #endif
 
-#if SDL_VERSION_ATLEAST(2, 0, 10)
-	SDL_SetHint(SDL_HINT_RENDER_BATCHING, "1");
-#endif
+	if (sdl2_runtime_version >= SDL_VERSIONNUM(2,0,10)) {
```

Comment:
@cmitu  This broke the coverity workflow — it also needs to check the build time SDL version, not just the runtime one, since otherwise SDL_HINT_RENDER_BATCHING may not be defined. That's the case on github, since SDL is at 2.0.8.

Reply:
Should be fixable. I see Ubuntu 18.04 LTS still uses SDL 2.0.8, so it's still a viable target - https://github.com/gemrb/gemrb/pull/939.